android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMPBrowserActivity.java

```diff
@@ -32,8 +32,8 @@
   private WebView webView;
   private static final String KMP_PRODUCTION_HOST = "https://keyman.com";
   private static final String KMP_STAGING_HOST = "https://staging-keyman-com.azurewebsites.net";
-  private static final String KMP_SEARCH_URL_FORMATSTR = "%s/keyboards%s?embed=android&version=%s";
-  private static final String KMP_LANGUAGE_FORMATSTR = "/languages/%s";
```

Comment:
This pattern still works for languages, e.g. `/keyboards/languages/am` -- no need to rewrite :)

Reply:
ok. I'll revert the language query and pivot this PR to only have the title bar tweak.